.github/pr_template_release.md

```diff
@@ -0,0 +1,46 @@
+## Release Checklist
+
+- [ ] **Canary Version Testing**
+
+  - [ ] I have thoroughly tested the release using a canary version: `v2...`
+  - [ ] The canary version has been verified to work as expected.
+  - [ ] All major features and changes have been tested and validated.
+
+- [ ] **Web3Modal Team Testing**
+
+  - [ ] The Web3Modal team has tested the release for compatibility and functionality.
+  - [ ] The package integrates seamlessly with Web3Modal (if relevant).
+  - [ ] Any reported issues or bugs have been addressed or documented.
+
+- [ ] **React Native Team Testing**
+
+  - [ ] The React Native team has tested the release for compatibility and functionality (if relevant).
+  - [ ] The package works correctly in React Native environments.
+  - [ ] Any reported issues or bugs have been addressed or documented (if applicable).
+
+## Description
+
+Provide a detailed description of the changes made in this release. Please include relevant information about new features, bug fixes, or improvements.
+
+If applicable, mention any specific areas that need additional attention during the review process.
+
+## Testing Instructions
+
+Outline the steps needed to test the release thoroughly. Include any specific scenarios or test cases that should be executed to verify the correctness and stability of the package. If necessary, provide links to additional resources, testing environments, or tools that can assist in the testing process.
+
+## Additional Notes
+
+Include any additional information, concerns, or considerations related to the release that might be important for the reviewers to be aware of.
+
+## Definition of Done
+
+- [ ] The release has been tested using a canary version.
+- [ ] The release has been reviewed and approved by the Web3Modal team (if relevant).
+- [ ] The release has been reviewed and approved by the React Native team (if relevant).
+- [ ] All necessary documentation, including API changes or new features, has been updated.
```

Comment:
we could add a section with related links for PRs with docs/api/other updates

Reply:
Added a "Related Issues and Pull Requests" section 👍